runtime/stdlib/contract_update_validation.go

```diff
@@ -215,6 +216,32 @@ func (validator *ContractUpdateValidator) hasErrors() bool {
 	return len(validator.errors) > 0
 }
 
+func collectRemovedTypePragmas(validator UpdateValidator, pragmas []*ast.PragmaDeclaration) map[string]struct{} {
+	removedTypes := map[string]struct{}{}
+
+	for _, pragma := range pragmas {
+		invocationExpression, isInvocation := pragma.Expression.(*ast.InvocationExpression)
+		if !isInvocation || len(invocationExpression.Arguments) != 1 {
```

Comment:
Maybe report a dedicated error for `len(invocationExpression.Arguments) != 1`. The current code will just silently ignore something like `#removedType(A, B)`. This would need to get moved after the check if the pragma is `removedType`.

Reply:
```suggestion
		if !isInvocation {
```

Can you please also add tests for the error reported below (`if len(invocationExpression.Arguments) != 1 {`), e.g. for the case where length is 0 or 2?